Sources/W3CTraceContext/SpanID.swift

```diff
@@ -51,45 +55,64 @@ public struct SpanID: Sendable {
     }
 
     /// An 8-byte array.
-    public typealias Bytes = (UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8)
+    public struct Bytes: Equatable, Hashable, Sendable {
+        @usableFromInline
+        let _bytes: (UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8)
+
+        public init(_ bytes: (UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8)) {
+            _bytes = bytes
+        }
+
+        public init(_ one: UInt8, _ two: UInt8, _ three: UInt8, _ four: UInt8, _ five: UInt8, _ six: UInt8, _ seven: UInt8, _ eight: UInt8) {
+            _bytes = (one, two, three, four, five, six, seven, eight)
+        }
```

Comment:
Do you think both of these initializers are necessary? I'd like to keep the public API to a minimum if possible.

Reply:
No, you are right, we should probably keep the public API at a minimum.

Reply:
Do you have a preference? IMO, the one which directly takes a tuple feels a bit cleaner.